channels/chan_usrp.c

```diff
@@ -593,6 +595,7 @@ static int usrp_xwrite(struct ast_channel *ast, struct ast_frame *frame)
 				remque((struct qelem *) qp);
 				ast_free(qp);
 			}
+			ast_log(LOG_WARNING, "Channel %s: Receive queue exceeds threshold\n", ast_channel_name(ast));
```

Comment:
Could we log the threshold value itself too here?
I know it's supposed to be static, but just in case somebody changes it

Reply:
I added the threshold to the message.